src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2315,6 +2377,97 @@
             "required": [ "name" ]
         },
 
+        "toolComponentTranslation": {
+            "description": "TBD.",
+            "additionalProperties": false,
+            "type": "object",
+            "properties": {
+
+                "toolComponentGuid": {
+                    "description": "TBD must be GUID format: matches toolComponent.guid.",
+                    "type": "string"
+                },
+
+                "location": {
+                    "description": "The location where translation can be obtained.",
```

Comment:
>where [](start = 49, length = 5)

"The location of the translation."

Sorry I can't seem to do better than that. I don't like "can be obtained" because that sounds like a download URI. In fact I just suggested to @michaelcfanning that we add a `string`-valued `downloadUri` to `toolComponentTranslation`, similar to the existing `toolComponent.downloadUri`, which we  describe as "The absolute URI from which the component can be downloaded" (so here, "... from which the translation can be downloaded").

But we haven't decided to do that yet.